source/ios/AdaptiveCards/ADCIOSVisualizer/TestFiles/Input.ChoiceSet.json

```diff
@@ -28,6 +28,28 @@
 				}
 			]
 		},
+        {
+            "type": "Input.ChoiceSet",
+            "id": "myColor1",
+            "style": "compact",
+            "isMultiSelect": false,
+            "placeholder": "Please choose",
```

Comment:
Spaces should be tabs. Also, question - is placeholder going to work with filtered choiceset?

Reply:
it's part of the test sample files. 
place holder does work, when the filtered list expanded, it will show the list of filtered list.